worker.go

```diff
@@ -0,0 +1,76 @@
+package rollrus
+
+import (
+	"fmt"
+	"os"
+	"sync"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/stvp/roll"
+)
+
+type job struct {
+	client roll.Client
+	entry  *log.Entry
+}
+
+func (j job) sendToRollbar() {
+	entry := j.entry
```

Comment:
should make a check for `if entry == nil`

Reply:
👍